src/com/project_phone_book/controller/TeleBookController.java

```diff
@@ -0,0 +1,116 @@
+package com.project_phone_book.controller;
+
+import com.project_phone_book.model.Contact;
+import com.project_phone_book.model.TeleBook;
+
+import java.io.File;
+import java.util.Scanner;
+
+public class TeleBookController {
+
+//    public static final int ADD_CONTACT = 0;
+//    public static final int FIND_BY_CHARS = 1;
+//    public static final int DELETE = 2;
+//    public static final int END = 3;
+
+    TeleBook teleBook = new TeleBook();
+    Scanner sc = new Scanner(System.in);
+    FileOperations file = new FileOperations();
+
+
+    public void phoneBookApi() {
+        createFileAndAddToList();
+        Option option = null;
+        while (option != Option.END) {
+            intro();
+            int choice = nextInt();
+            try {
+                option = Option.values()[choice];
+                switch (option) {
+                    case ADD_CONTACT:
+                        addContact();
+                        break;
+                    case FIND_BY_CHARS:
+                        findByChars();
+                        break;
+                    case DELETE:
+                        delete();
+                        break;
+                    case PRINT_ALL_CONTACTS:
+                        printAll();
+                        break;
+                    case END:
+                        end();
+                        break;
+                    default:
+                        System.out.println("You choose wrong option.");
+                }
+            } catch (ArrayIndexOutOfBoundsException e) {
+                System.out.println("You took incorrect option.");
+            } catch (NullPointerException e) {
+                System.out.println("THere is no option such this.");
```

Comment:
you have three exception printing pretty much same message. Why don't you use one catch for all of them?